packages/manager/apps/container/src/container/common/urls-constants.ts

```diff
@@ -69,6 +70,7 @@ export function useURL(environment: Environment): UseURL {
       const user = environment.getUser();
       const regionURL = urls[region];
       const url = regionURL[id];
+      if(!url) return;
```

Comment:
i had to modify this file because, if the region is US, the URL will be null and would have thrown an exception at line 74 (....url[user.ovhSubsidiary])